x-pack/plugins/security_solution/server/lib/telemetry/event_based/events.ts

```diff
@@ -123,6 +123,86 @@ export const ASSET_CRITICALITY_SYSTEM_PROCESSED_ASSIGNMENT_FILE_EVENT: EventType
     },
   };
 
+export const ALERT_SUPPRESSION_EVENT: EventTypeOpts<{
+  suppressionAlertsCreated: number;
+  suppressionAlertsSuppressed: number;
+  suppressionRuleName: string;
+  suppressionDuration: number;
+  suppressionGroupByFieldsNumber: number;
+  suppressionGroupByFields: string[];
+  suppressionRuleType: string;
+  suppressionMissingFields: boolean;
+  suppressionRuleId: string;
+}> = {
+  eventType: 'alert_suppression_on_rule_execution',
+  schema: {
+    suppressionAlertsCreated: {
+      type: 'long',
+      _meta: {
+        description:
+          'Number of alerts created during rule execution with configured alert suppression',
+      },
+    },
+    suppressionAlertsSuppressed: {
```

Comment:
Can you give example what this field represent?

In my test I have query rule with 10 events, which generate 1 alert.

This field has 0 in the end, is it correct?

Reply:
I have this in description for the field
```
          'Number of alerts suppressed during rule execution with configured alert suppression',
```

So, that is basically suppressed alerts. It should represent number of suppressed alerts.

If your 1 generated alert has 0 suppressed - so it would be 0.
Check suppression field `kibana.alert.suppression.docs_count` of your alert. If there are many alerts - that number would be a sum of all suppressed alerts

Reply:
So, `docs_count` and `suppressionAlertsSuppressed ` should be the same? For me it's currently not the same:
<img width="1447" alt="Screenshot 2024-08-05 at 11 19 28" src="https://github.com/user-attachments/assets/96a08abe-4f68-4b83-8629-a0f41ed6f206">
<img width="869" alt="Screenshot 2024-08-05 at 11 18 51" src="https://github.com/user-attachments/assets/88e58eb5-6dbf-4bef-97d5-8f83ee69f5c7">

Reply:
@nkhristinin , fixed in https://github.com/elastic/kibana/pull/188753/commits/1e2680661342ae22f3a68eda765770c561d0d518